client/landing/subscriptions/components/comment-list/comment-list.tsx

```diff
@@ -1,35 +1,100 @@
+import {
+	CellMeasurer,
+	CellMeasurerCache,
+	List,
+	WindowScroller,
+} from '@automattic/react-virtualized';
+import { useCallback, useRef } from '@wordpress/element';
 import { useTranslate } from 'i18n-calypso';
+import withDimensions from 'calypso/lib/with-dimensions';
 import CommentRow from './comment-row';
 import './styles.scss';
 import type { PostSubscription } from '@automattic/data-stores/src/reader/types';
 
 type CommentListProps = {
 	posts?: PostSubscription[];
+	width?: number;
 };
 
-const CommentList = ( { posts }: CommentListProps ) => {
+const cellMeasureCache = new CellMeasurerCache( {
+	fixedWidth: true,
+	// Since all our rows are of equal height, we can use this performance optimization
+	keyMapper: () => 1,
+} );
+
+const CommentList = ( { posts, width }: CommentListProps ) => {
 	const translate = useTranslate();
+	const windowScrollerRef = useRef();
+
+	const rowRenderer = useCallback(
```

Comment:
What do you think on moving this implementation into a new VirtualizedList component? It could receive the `items` and the inner part of this `rowRenderer` function `(item, forwardedRef, style) => Row`. That way we would isolate this implementation and make it easier to apply on `SiteList` later.

Reply:
I added it to a separate component now 🥳